src/transformers/configuration_utils.py

```diff
@@ -1050,6 +1069,15 @@ def register_for_auto_class(cls, auto_class="AutoConfig"):
 
         cls._auto_class = auto_class
 
+    def has_set_generation_parameters(self) -> bool:
```

Comment:
Can we make this a private method? I don't think it makes sense to expose this to users - as it's a helper for BC